lib/rangeclient.rb

```diff
@@ -37,7 +37,7 @@ def initialize(options = {})
   end
 
   def expand(arg)
-    escaped_arg = URI.escape arg
+    escaped_arg = ERB::Util.url_encode arg
```

Comment:
Would prefer we use `URI.encode_www_form_component` here.

Reply:
Done.